FreshRSS/FreshRSS
# Fixed: No user query - no information

Before:
![grafik](https://github.com/user-attachments/assets/3414cc7e-f9a9-4871-b485-dd53707f10da)

After:
![grafik](https://github.com/user-attachments/assets/f6043de9-8f15-42ef-85be-2e64370426ee)



How to test the feature manually:

1. got to user query config page
2. have 0 user queries
3. see the screen

Pull request checklist:
